modules/bar/media/index.ts

```diff
@@ -76,12 +83,17 @@ const Media = () => {
                             label: songIcon.bind("value"),
                             maxWidthChars: 30,
                         }),
-                        Widget.Label({
+                        Widget.Box({
                             class_name: "bar-button-label media",
-                            label,
-                            truncate: "end",
-                            wrap: true,
-                            maxWidthChars: 30,
+                            child: Utils.merge(
+                                [truncation.bind("value"), truncation_size.bind("value")], 
```

Comment:
Should we move these bindings to the mediaLabel so they're all consolidated? Then we can handle the truncation in that `Utils.watch` as well. That way we don't need to nest our Widget.Label in a Widget.Box and we can just call the label as:
```
Widget.Label({
  label: mediaLabel
}))
```